### PR TITLE
dockerise

### PR DESCRIPTION
**Pull request by kaisbaccour**

### CODE REVIEW

**Review by 0xantman — APPROVED** [review]
Good 👍

File: index.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ global.AbortController = AbortController
     CosmJS
  */
 const mnemonic = process.env.MNEMONIC;
+const DENOM = process.env.DENOM;
```
Comment (webmaster128) [inline index.js:18]: let's use lower camelCase for JavaScript variables for consistency

```suggestion
const denom = process.env.DENOM;
```

File: index.js [hunk 2]
```diff
@@ -15,12 +15,12 @@ global.AbortController = AbortController
     CosmJS
  */
 const mnemonic = process.env.MNEMONIC;
-console.log(process.env.MNEMONIC)
-const wallet = await DirectSecp256k1HdWallet.fromMnemonic(mnemonic);
+const DENOM = process.env.DENOM;
+const prefix = {prefix: process.env.PREFIX}
```
Comment (webmaster128) [inline index.js:19]:
```suggestion
const options = {prefix: process.env.PREFIX}
```

File: index.js [hunk 3]
```diff
@@ -15,12 +15,12 @@ global.AbortController = AbortController
...
+const wallet = await DirectSecp256k1HdWallet.fromMnemonic(mnemonic, prefix);
 const [firstAccount] = await wallet.getAccounts();
-
 const rpcEndpoint = process.env.ENDPOINT;
-const signer = await SigningCosmWasmClient.connectWithSigner(rpcEndpoint, wallet);
+const signer = await SigningCosmWasmClient.connectWithSigner(rpcEndpoint, wallet, prefix);
```
Comment (webmaster128) [inline index.js:23]:
```suggestion
const signer = await SigningCosmWasmClient.connectWithSigner(rpcEndpoint, wallet, options);
```

File: index.js (hunk 2 above)
Comment (0xantman) [reply index.js:19]: We already have this variable called options 
https://github.com/noislabs/nois-bot/blob/9bf791bd87dd3e9c3d3e24fc37df18cb17f0fcd0/index.js#L39

Comment (webmaster128) [reply index.js:19]: Ah okay, I see. Yeah, not super important. It's just an object that does more than only prefix (we don't use the other options yet.

Comment (0xantman) [reply index.js:19]: Let’s call it chain_options and drand_options

Comment (webmaster128) [reply index.js:19]: Yes!

File: index.js [hunk 4]
```diff
@@ -36,6 +36,7 @@ const urls = [
 ]
 
 async function start (){
+
     const options = { chainHash }
```
Comment (0xantman) [inline index.js:40]:
```suggestion
    const drand_options = { chainHash }
```

**Pull request merged**